orderbook/src/account_balances.rs

```diff
@@ -216,14 +265,19 @@ mod tests {
         let settlement = contracts::GPv2Settlement::deployed(&web3).await.unwrap();
         let allowance = settlement.allowance_manager().call().await.unwrap();
         let fetcher = Web3BalanceFetcher::new(web3, allowance, settlement.address());
-        let owner = H160(hex!("c978f4364c03a00352e8c7d9619b42e26b6424ab"));
-        let token = H160(hex!("c12d1c73ee7dc3615ba4e37e4abfdbddfa38907e"));
+        let owner = H160(hex!("78045485dc4ad96f60937dad4b01b118958761ae"));
```

Comment:
Not sure where this address is coming from, but it looks like the token itself has a balance of its own token. This is likely not going to move (whereas this might transfer out its balance at some point)?